### ISSUE TITLE
Remove master/slave terminology in code when possible & replace with updated language

### ISSUE THREAD

**Issue by NuclearMonster** [ioquake/ioq3#545]
We started by renaming our code branches, lets keep going and change master/slave in every instance in the code when it is possible to do so and retain compatibility. When it isn't possible, we can at least comment that in the code.

https://ioquake3.org/ioquake3/ioquake3-git-branch-renamed-to-main/

**Please note: anyone arguing against this proposal in this thread or otherwise will be banned from this repository. The only policy for racists is a 0 tolerance policy.**

**Comment by UInt2048** [comment]
I agree with the proposal. However, does giving a reaction to this proposal actually count as arguing either for or against it? It seems a little strange considering that the reacting user might just be not in favor of the no second chances part of the ban policy.

**Comment by UInt2048** [comment]
I agree with the proposal. However, does giving a reaction to this proposal actually count as arguing either for or against it? It seems a little strange considering that the reacting user might just be not in favor of the no second chances part of the ban policy.

Edit: For the record, I have not given any reaction on the proposal yet.

**Comment by UInt2048** [comment]
As an addendum, to those who are reading this and thinking that this won't do anything to fight racism, nobody said it would! Why are everyday people across the globe showing their solidarity with Ukraine right now during the Russian invasion of Ukraine? Do they think it's going to help fight Putin? Of course not!

**Comment by ellisgl** [comment]
I'm totally in favor of it. I would still support the project if they decided to move to GitLab or another git hosting solution.

**Comment by fra9000** [comment]
The populist reaction to american police shooting black people has brought the Hardware hard disk to be renamed from master to main..they deserved that. Bad hard disk!. Git repository..you racist master now you will be changed to main. And the funny thing is that both HW and SW have never been related to White Master or Black Slave. Nevertheless US police is still killing poor people in the street. But yeah many politically correct commits to avoid screaming populists yelling around. How funny the human kind is getting...
Just an opinion from an antiracist european.

**Comment by NuclearMonster** [comment]
This is not the place to discuss weather you are for or against this change and when random people show up who do not have anything to do with this project to discuss it, your comments will be deleted from this thread. This is not the debate zone. This is the place for getting things done.

**Comment by TheAwesome98-Real** [comment]
> This is not the place to discuss whether you are for or against this change

ok so where is that
